src/main/java/com/cburch/logisim/fpga/hdlgenerator/ToplevelHDLGeneratorFactory.java

```diff
@@ -59,17 +57,17 @@
   private final HashMap<String, Boolean> LedArrayTypesUsed; 
 
 
-  public ToplevelHDLGeneratorFactory(
-      long FPGAClock, double TickClock, Circuit TopLevel, MappableResourcesContainer IOComponents) {
+  public ToplevelHDLGeneratorFactory(long FPGAClock, double TickClock, Circuit TopLevel, 
```

Comment:
variable names should start with lowercase. Please rename.

Reply:
I agree with your comment, but I think that in this case we should leave it as is, because in the whole logisim.fpga.* I consistently used in the passed capitals for the first letter of a variable. I think that we should generate an other PR where I do a complete code refactoring of logisim.fpga.* for var and variable names. What do you think?

Reply:
I reworked this class completely, but there are a lot more in the other classes.